src/components/CloneIssuesButton.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useLocation } from "@docusaurus/router";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+
+const SignInButton = () => {
+  const { siteConfig } = useDocusaurusContext();
+  const location = useLocation();
+
+  const qs = new URLSearchParams({
+    client_id: siteConfig.customFields.CLIENT_ID,
+    state: JSON.stringify({
+      prevPath: location.pathname,
+    }),
+  });
+
+  const url = `https://github.com/login/oauth/authorize?${qs}`;
+
+  return (
+    <a href={url} className="button button--primary">
+      Sign In
+    </a>
+  );
+};
+
+const CloneButton = ({ repo, issue, token }) => {
+  const [loading, setLoading] = React.useState(false);
+  const [error, setError] = React.useState(false);
+  const [success, setSuccess] = React.useState(false);
+
+  const clone = () => {
+    setLoading(true);
+    const url = issue
+      ? `${API_URL}/clone?repo=${repo}&issue=${issue}`
+      : `${API_URL}/clone?repo=${repo}`;
+
+    fetch(url, {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        setLoading(false);
+        setSuccess(data.message);
+      })
+      .catch((err) => {
+        setLoading(false);
+        setError(err);
+        console.log(err);
+      });
+  };
+
+  if (error) {
+    return <strong>Something went wrong! See console for details.</strong>;
```

Comment:
My worry here is that `<strong>` is perhaps not obvious enough for users to notice? Perhaps we could use the [Alert](https://infima.dev/docs/components/alert) styling?